### PR TITLE
Autodiscover `*.tiff` files in addition to `*.tif`

### PR DESCRIPTION
**Pull request by sharkAndshark**
Autodiscover `*.tiff` files in addition to `*.tif`

- [x] Autodiscover `*.tiff` files in addition to `*.tif`
- [x] Update test
- [x] Update doc

### CODE REVIEW

File: docs/src/sources-cog-files.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ Martin can also serve raster sources like local [COG(Cloud Optimized GeoTIFF)](h
 ```bash
 # Configured with a directory containing TIFF files.
```
Comment (nyurik) [inline docs/src/sources-cog-files.md]:
```suggestion
# Configured with a directory containing `*.tif` or `*.tiff` TIFF files.
```

File: docs/src/sources-cog-files.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ Martin can also serve raster sources like local [COG(Cloud Optimized GeoTIFF)](h
...
 martin /with/tiff/dir1 /with/tiff/dir2
-# Configured with dedicated TIFF file.
-martin /path/to/target1.tif /path/to/target1.tif
+# Configured with dedicated TIFF file, `.tiff` works either
```
Comment (nyurik) [inline docs/src/sources-cog-files.md]:
```suggestion
# Configured with dedicated TIFF file
```

File: martin/src/args/root.rs [hunk 3]
```diff
@@ -275,4 +280,15 @@ mod tests {
...
+
+    #[test]
+    fn cli_multiple_extensions() {
+        let args = Args::parse_from(["martin", "../tests/fixtures/cog"]);
+
+        let env = FauxEnv::default();
+        let mut config = Config::default();
+        let err = args.merge_into_config(&mut config, &env);
+        println!("{:?}", err);
+        println!("{:?}", config);
```
Comment (nyurik) [inline martin/src/args/root.rs]:
```suggestion
        println!("{err:?}");
        println!("{config:?}");
```

**Review by CommanderStorm — APPROVED** [review]
Looks perfect ♥️
=> merge at your command

**Pull request merged**